canonical-websites/insights.ubuntu.com
# Bump global nav version

## Done

Bump global nav version

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- Make sure navigation has a blog link and not an insights link


## Issue / Card

Fixes [#512](https://github.com/ubuntudesign/web-squad/issues/512)
